### PR TITLE
Second set of tests for the More info card

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Follow up for #201

### CODE REVIEW

File: tests/test_addon_detail.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def test_non_promoted_addon(selenium, base_url, variables):
     # checks the presence of an install warning
     assert addon.install_warning.is_displayed()
     assert variables['install_warning_message'] in \
-        addon.install_warning_message
+           addon.install_warning_message
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:74]: This line looks spaced a bit far, also it looks  like a tab with a couple spaces See: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

File: tests/test_addon_detail.py [hunk 2]
```diff
@@ -286,7 +286,7 @@ def test_more_info_addon_size(selenium, base_url, variables):
     # get the file URL and read its size - conversion from bytes to Mb is required
     file = urllib.request.urlopen(
         'https://addons.allizom.org/firefox/downloads/file/1097275/ghostery_privacy_ad_blocker-8.5.6-an+fx.xpi')
-    size = file.length/(1024*1024)
+    size = file.length / (1024 * 1024)
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:289]: I don't see the need for this change. Any reason why you added these spaces?

File: tests/test_addon_detail.py [hunk 3]
```diff
@@ -310,3 +310,64 @@ def test_more_info_external_license(selenium, base_url, variables):
...
+    # and not all add-ons will have it
+    try:
+        privacy = addon.more_info.addon_privacy_policy()
+        # checks that the AMO privacy policy page opens and has the correct content
+        assert 'Privacy policy for Ghostery - Privacy Ad Blocker' in \
+               privacy.custom_licence_and_privacy_header
+        assert privacy.custom_licence_and_privacy_text.is_displayed()
+        assert privacy.custom_licence_and_privacy_summary_card.is_displayed()
+    except NoSuchElementException:
+        print("The add-on doesn't have a privacy policy")
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:341]: If this test passes pytest may not print this line. I think this test should be split. One for testing an addon with a privacy policy and one for an addon without a policy

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Looking gooood

File: tests/test_addon_detail.py (hunk 3 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py]: I've added a separate test for add-ons without a Privacy Policy for better coverage. I did the same for the eula test.

File: tests/test_addon_detail.py (hunk 2 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py]: Spaces removed. I think they have been added by the automatic formatter used by my IDE.

**Review by jrbenny35 — APPROVED** [review]
Great work once again!

**Pull request merged**